infra/base-images/base-builder/compile

```diff
@@ -115,7 +119,7 @@ BUILD_CMD="bash -eux $SRC/build.sh"
 
 # We need to preserve source code files for generating a code coverage report.
 # We need exact files that were compiled, so copy both $SRC and $WORK dirs.
-COPY_SOURCES_CMD="cp -rL --parents $SRC $WORK /usr/include /usr/local/include $GOPATH $OUT"
+COPY_SOURCES_CMD="cp -rL --parents $SRC $WORK /usr/include /usr/local/include $GOPATH /rust $OUT"
```

Comment:
will this fail if /rust does not exist?

Reply:
Likely yes
Should I use an environment variable like `$GOPATH` and set it appropriately ?

Reply:
yes right.

Reply:
ok